mne/decoding/ssd.py

```diff
@@ -253,12 +251,11 @@ def get_spectral_ratio(self, ssd_sources):
         sorter_spec = spec_ratio.argsort()[::-1]
         return spec_ratio, sorter_spec
 
-    def apply(self):
-        """Not implemented, see ssd.inverse_transform() instead."""
-        # Exists because of _XdawnTransformer
-        raise NotImplementedError('See ssd.inverse_transform()')
+    def inverse_transform(self):
+        """Not implemented yet."""
+        raise NotImplementedError('inverse_transform is not yet available.')
 
-    def inverse_transform(self, X):
+    def apply_array(self, X):
```

Comment:
why not just apply?

Reply:
Well apply supports classically mne objects, not arrays. So apply would raise the expectation that it accepts instances.